funnel/models/video.py

```diff
@@ -1,6 +1,7 @@
 import urllib.parse
 
 from pytz import utc
+from simplejson import JSONDecodeError
```

Comment:
`requests` uses simplejson?

Reply:
It uses `simplejson` if available, or else `json`.